src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,38 @@
+package main.java.pages;
+
+import main.java.utils.DriverWeb;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import test.java.BaseTest;
+
+public class LoginPage {
+
+    WebDriver driver = DriverWeb.get();
+
+    WebElement textLogin = driver.findElement(By.className("blockHeader__block-header--AHdxL"));
+    WebElement inputUserName = driver.findElement(By.name("login"));
+    WebElement inputPassword = driver.findElement(By.name("password"));
+    WebElement buttonLogin = driver.findElement(By.tagName("button"));
+
+    public void verifyLoginPageOpened() {
+
+        //BaseTest.logger.info("Verifying login page title text visibility");
```

Comment:
It's better nor keep commented code in repo. Feel free to remove, if it's not required.